apps/cheshire-cat-ai/config.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "Cheshire Cat AI",
+    "available": true,
+    "port": 1865,
+    "exposable": true,
+    "id": "cheshire-cat-ai",
+    "description": "The Cheshire Cat is an open-source, hackable and production-ready framework that allows developing intelligent personal AI assistant agents on top of Large Language Models (LLM).",
+    "tipi_version": 1,
+    "version": "latest",
+    "categories": ["ai"],
+    "short_desc": "A production-ready AI framework that empowers individuals with elementary Python knowledge to save 2.000 of hours in development time.",
```

Comment:
The short description should be... shorter.

Reply:
I made it as shorter as I can 😄